community/0-welcome.mdx

```diff
@@ -14,27 +14,28 @@ Please create an issue for the following:
 
 - [Bug Reports](https://github.com/microsoft/playwright/issues/new?assignees=&labels=&template=bug.md&title=%5BBUG%5D)
 - [Feature Requests](https://github.com/microsoft/playwright/issues/new?assignees=&labels=&template=feature_request.md&title=%5BFeature%5D)
-- [I have a question](https://github.com/microsoft/playwright/issues/new?assignees=&labels=&template=question.md&title=%5BQuestion%5D)
 - [Report Regression](https://github.com/microsoft/playwright/issues/new?assignees=&labels=&template=regression.md&title=%5BREGRESSION%5D%3A+)
 - [Report a security vulnerability](https://github.com/microsoft/playwright/security/policy)
 
-Or why not join our [GitHub Discussions community](https://github.com/microsoft/playwright/discussions)
-
 ## Contributing
 
 Check out our [contributing guide](https://github.com/microsoft/playwright/blob/main/CONTRIBUTING.md) if you would like to contribute to Playwright.
 
 ## Community Discord
 
-Join our community [Discord Server](https://aka.ms/playwright/discord) to connect with other developers using Playwright.
+Join our community [Discord Server](https://aka.ms/playwright/discord) to connect with other developers using Playwright, ask questions in our 'help-playwright' forum, share your articles and videos and join live events on the Playwright stage.
```

Comment:
we could link here to the help-playwright forum, I think thats a direct link: https://discord.com/channels/807756831384403968/1054804523652231198